backend/app/alembic/versions/2026_02_18_2104-5934c2bdf3e1_add_item_table_objective_category_and_.py

```
@@ -0,0 +1,54 @@
+"""add_item_table_objective_category_and_rewardtype_enums
+
+Revision ID: 5934c2bdf3e1
+Revises: add_medical_storage_fields
+Create Date: 2026-02-18 21:04:16.052683
+
+"""
+
+from typing import Sequence, Union
```

Ruff (0.15.1)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)